workloads/dataset-controller/README.md

```diff
@@ -0,0 +1,33 @@
+== Dataset Controller ==
+
+A simple controller for `datasets.new.artificial.cloud` custom objects.
```

Comment:
artificial.cloud -> artificialwisdom.cloud

why `new`?

Reply:
I read an article about how `new` is better than `unstable` for conveying expectations. But it's the same idea as the standard `unstable`, `stable`, etc. namespacing.

Reply:
by using `new`, you mean encoding semantic information about feature or API lifecycle into the object name?